mods/ra/maps/soviet-09/soviet09-AI.lua

```diff
@@ -0,0 +1,190 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+IdleHunt = function(unit) if not unit.IsDead then Trigger.OnIdle(unit, unit.Hunt) end end
+
+AttackGroup = { }
+AttackGroupSize = 12
+AlliedInfantry = { "e1", "e3" }
+AlliedVehicles = { "jeep", "1tnk", "2tnk", "2tnk" }
+AlliedAircraftType = { "heli" }
+Longbows = { }
+
+ProductionInterval =
+{
+	easy = DateTime.Seconds(25),
+	normal = DateTime.Seconds(15),
+	hard = DateTime.Seconds(5)
+}
+AttackPaths =
+{
+	{ SouthAttack1.Location, SouthAttack2.Location, DefaultCameraPosition.Location },
+	{ TruckStop2.Location, TruckStop1.Location, DefaultCameraPosition.Location }
+}
+
+WTransUnits = { { "2tnk", "1tnk", "1tnk", "e3", "e3" }, { "2tnk", "2tnk", "2tnk"  } }
+WTransDelays =
+{
+	easy = 5,
```

Comment:
I suppose we want to do the same here as in `08` and define the minutes directly.